deploy.go

```diff
@@ -194,6 +196,28 @@ func RunDeploy(c *cli.Context) error {
 	return nil
 }
 
+func displayNewCommits(owner string, repo string, c *cli.Context, client *github.Client) {
+	ref := Ref(c.String("ref"), git.Head)
+
+	opt := &github.DeploymentsListOptions{
+		Environment: c.String("env"),
+	}
+
+	deployments, _, error := client.Repositories.ListDeployments(owner, repo, opt)
+	if error == nil {
+		sha := *deployments[0].SHA
+		compare, _, cmp_error := client.Repositories.CompareCommits(owner, repo, sha, ref)
```

Comment:
It's a little more idiomatic to just re-use the `err` variable instead of `cmp_error` (also, use camel case for variable names in go). Also missing `err != nil` case.